crits/samples/handlers.py

```diff
@@ -1067,6 +1093,11 @@ def handle_file(filename, data, source, method='Generic', reference=None, relate
                                      "data itself, need to be supplied.")
                 retVal['success'] = False
 
+            if sha1_digest:
+                sample.sha1 = sha1_digest
+            if sha256_digest:
+                sample.sha256 = sha256_digest
```

Comment:
Just checking since the hashes are also handled further up and wasn't sure why the mimetype and size weren't also added here.